pkg/kubectl/bind-apiservice/plugin/servicebindings_test.go

```diff
@@ -0,0 +1,756 @@
+/*
+Copyright 2022 The Kube Bind Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	kubebindv1alpha1 "github.com/kube-bind/kube-bind/pkg/apis/kubebind/v1alpha1"
+)
+
+func TestHumanReadablePromt(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name           string
+		testData       kubebindv1alpha1.PermissionClaim
+		expectedOutput string
+	}{
+		{"Owner=Provider",
```

Comment:
**issue** Missing test cases for donate and adopt without selector e.g.
```go
		{"Selector={},donate=true",
			kubebindv1alpha1.PermissionClaim{
				GroupResource: kubebindv1alpha1.GroupResource{
					Group:    "",
					Resource: "foo",
				},
				Version:  "v1",
				Selector: kubebindv1alpha1.ResourceSelector{},
				Required: true,
				Create: &kubebindv1alpha1.CreateOptions{
					Donate: true,
				},
			},
			"Create and modify user-owned foo objects (apiVersion: \"v1\") on your cluster.\n" +
				"The provider will not be able to modify or delete said objects.\n" +
				"Accepting this Permission is required in order to proceed.\n" +
				"Do you accept this Permission? [No,Yes]\n",
		},
		{"Selector={},adopt=true,update.preserving=[\"spec\"]",
			kubebindv1alpha1.PermissionClaim{
				GroupResource: kubebindv1alpha1.GroupResource{
					Group:    "",
					Resource: "foo",
				},
				Version:  "v1",
				Selector: kubebindv1alpha1.ResourceSelector{},
				Adopt:    true,
				Update: &kubebindv1alpha1.UpdateOptions{
					Preserving: []string{"spec"},
				},
			},
			"The provider wants to become owner of all user-created foo objects (apiVersion: \"v1\").\n" +
				"The provider will be able to access, modify and delete said objects on your cluster.\n" +
				"The following fields of the objects will still be managed by the user:\n" +
				"\t\"spec\"\n" +
				"Accepting this Permission is optional.\n" +
				"Do you accept this Permission? [No,Yes]\n",
		},
```

Reply:
I added (more) test cases with an empty selector in 926bb5993775516f6dbfe28787228a9b11d95f92. I think i should now cover all cases.